README.md

```diff
@@ -161,7 +147,7 @@ Use the `model/predict` endpoint to load a test file and get captions for the im
 
 You can also test it on the command line, for example:
 
-    curl -F "image=@assets/surfing.jpg" -X POST http://localhost:5000/model/predict
+    curl -F "image=@path/to/image.jpg" -X POST http://localhost:5000/model/predict
```

Comment:
My only thought here is that, since the model repo was not checked out, the test assets are not readily available. But we still have the JSON response for one of them below.

Perhaps we can say "Use the `model/predict` endpoint to get captions for an image from the API. The `[model assets folder](link)` contains a number of images you can use to test out the API."

Or something like that?

Reply:
That's why I changed it to an obviously fake path instead of the asset that isn't there anymore. @djalova and I discussed this over slack and we didn't want to remove the "example" part from the code pattern since it's a landing point for more than just MAX.

Reply:
Sure - I think it just ties the example output to a specific example input and we could just mention the model repo as a location of some useful test image.. But not a huge deal either way.

Reply:
@stevemart @rhagarty in terms of this as a code pattern what do you guys think?

Reply:
@ajbozarth i am OK with the obviously fake path, but i would suggest adding a comment under the curl command, something like ...

> See https://github.com/IBM/MAX-Image-Caption-Generator/tree/master/assets for sample images to test with, or supply your own

Reply:
Thanks @stevemart  I'll add that line, but do you think we should leave in the following example output still?

Reply:
@MLnick @stevemart I added a clarifying section with a link to the asset folder based on your text suggestions if you could check it. After adding it though I'm unsure whether we should remove or leave in the sample json output, what do each of you think?

Reply:
@ajbozarth i think it's fine to include the sample output

Reply:
thanks @stevemart 
Once @MLnick gives the ok on this I'll merge it
Though I do have one issue, does the new line between the curl and it's output break the flow of understanding? It just feels off to me. Should we move that note after the output or maybe somewhere before the curl command? Or should we add a line specifying that the output block is what output may look like?

Reply:
I like the wording of the paragraph now, but I agree the placement looks odd. I suggest moving the paragraph just above the line "You can also test it on the command line, for example:"

Reply:
I'll try moving it

Reply:
I moved it and it looks better there as you thought.